tests/jballerina-unit-test/src/test/resources/test-src/statements/arrays/sealed_array_listexpr_negative.bal

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+type Person abstract object {
+    public int age;
+    function getFullName() returns string;
+};
+
+// [0] cannot initialize abstract objects
+function createAbstractObjectArray() {
+    Person[5] x = [];
+    Person p1 = x[2];
+    p1.age = 20;
+}
+
+// [1] cannot create multi dimensional arrays of abstract objects
+function createAbstractObjectEmptyArray() {
+    Person[5][] y = [];
```

Comment:
This shouldn't fail right?

`Person[5][]` means 5 arrays of open Person arrays (`Person[]`)? So for the members, `[]` is a valid constructor?

Reply:
Fixed by https://github.com/ballerina-platform/ballerina-lang/pull/20749/commits/a40950b3d01179e69e9b8985cfb7bfcf277b393a